ui/src/saros/stf/server/rmi/htmlbot/impl/HTMLBotImpl.java

```diff
@@ -36,13 +36,13 @@ public IRemoteHTMLView view(View view) {
 
   @Override
   public List<String> getAccountList() throws RemoteException {
-    return BotUtils.getListItemsText(getBrowser(MainPage.class), SELECTOR_ACCOUNT_ENTRY);
+    return new JQueryHelper(getBrowser(MainPage.class)).getListItemsText(SELECTOR_ACCOUNT_ENTRY);
```

Comment:
How often is this called and can the browser for `MainPage.class` ever change? Would it be feasible/sensible to instantiate the `JQueryHelper` in the CTOR and hold it as an instance variable instead of always creating new instances?

Reply:
The browser could change (based on the functionality of `BrowserManager`). I am not really sure why the browser of a page should change. I think this is a topic we should keep in mind. Probably, the replacement of a browser is just a function which is never necessary (currently the browser is set only once in `BrowserCreator`) and just increases the complexity.

Reply:
Moving the instantiation into the CTOR might cause issues in the feature. Therefore I would prefer not to change it. What do you think?

Reply:
Fine with me. This is only a small optimization anyway.